### DISCUSSION TITLE
Outbound link tracking

### DISCUSSION THREAD

**Discussion by d19dotca** [umami-software/umami#1140] [Ideas]
Would be great to see Outbound link tracking included in Umami, similar to Matomo & Plausible does it. This way we can determine what links are being clicked to outside domains / websites.

Ref: 
* https://plausible.io/docs/outbound-link-click-tracking
* https://matomo.org/faq/new-to-piwik/faq_71/

**Comment by harrison-broadbent** [comment]
I would like to second this. 

This feature is the only reason I switched away from Umami to Pirsch — Pirsch has an extended script which can track outbound links, file downloads etc — https://docs.pirsch.io/advanced/outbound-links

If Umami added this I would probably switch back!
